ibis/backends/tests/test_temporal.py

```diff
@@ -41,17 +41,19 @@
 )
 from ibis.common.annotations import ValidationError
 
+mark_druid_timestamp_is_string = pytest.mark.notimpl(
```

Comment:
I also factored out this mark that was used about 50 times in this file. I could keep this out of this PR too.

Reply:
nvm this was made irrelevant by #9692